release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1102,6 +1102,73 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+// TODO: This known issue should be removed when RHCOS images begin using RHEL 9. RHEL 9 prevents this issue from occurring.
+* Intermittently, an IBM Cloud VPC cluster might fail to install because some worker machines do not start. Rather, these worker machines remain in the `Provisioned` phase. There is a workaround for this issue. From the host where you performed the initial installation, delete the failed machines and run the installation program again. (link:https://issues.redhat.com/browse/OCPBUGS-1327[*OCPBUGS#1327*])
```

Comment:
Line break might make this easier to parse, and suggest moving the link to the end (as with the previous bulleted item) so users don't misinterpret that link as the end of the bullet.

```suggestion
* Intermittently, an IBM Cloud VPC cluster might fail to install because some worker machines do not start. Rather, these worker machines remain in the `Provisioned` phase. 
+
There is a workaround for this issue. From the host where you performed the initial installation, delete the failed machines and run the installation program again.
```

Reply:
Yep. Agreed.